contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -189,3 +204,26 @@ func GetBlockGasLimit(header *types.Header, state vm.StateDB) (uint64, error) {
 	}
 	return gasLimit.Uint64(), nil
 }
+
+func GetLookbackWindow(header *types.Header, state vm.StateDB) (uint64, error) {
+	var lookbackWindow *big.Int
+	_, err := contract_comm.MakeStaticCall(
+		params.BlockchainParametersRegistryId,
+		blockchainParametersABI,
+		"getUptimeLookbackWindow",
+		[]interface{}{},
+		&lookbackWindow,
+		params.MaxGasForReadBlockchainParameter,
+		header,
+		state,
+	)
+	if err != nil {
+		if err == errors.ErrRegistryContractNotDeployed {
+			log.Debug("Error obtaining lookback window", "err", err, "contract", hexutil.Encode(params.BlockchainParametersRegistryId[:]))
+		} else {
+			log.Warn("Error obtaining lookback window", "err", err, "contract", hexutil.Encode(params.BlockchainParametersRegistryId[:]))
+		}
+		return 0, err
+	}
+	return lookbackWindow.Uint64(), nil
```

Comment:
this will have the side effect of truncating any 256-bit value to a go uint64?

Reply:
basically I'm double-checking that an invalid contract state won't cause a panic or something here